devtools/platform-descriptor-json/src/main/resources/codestarts/quarkus/core/buildtool/maven/codestart.yml

```diff
@@ -25,6 +25,7 @@ language:
         cmd:
           dev: mvn compile quarkus:dev
           package: mvn package
-          package-uberjar:  mvn package -PuberJar
```

Comment:
shouldn't all the refereences to mvn be `./mvn`

Reply:
ooh this is when no mvn wrapper....you need to check if the setup java gives you the right maven/gradle version